lizard_neerslagradar/tests/test_layers.py

```diff
@@ -29,3 +31,46 @@ def test_grid_name(self):
         adapter = layers.NeerslagRadarAdapter(None)
         self.assertEquals(adapter._grid_name('Nieuwegein', (1, 2)),
                           'Neerslag in Nieuwegein')
+
+
+class TestGetGraphValues(TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestCase, self).__init__(*args, **kwargs)
```

Comment:
dit kan je ook in de `def setUp(self):` doen, dan hoef je de init niet te overschrijven.